homeassistant/components/smhi/__init__.py

```diff
@@ -0,0 +1,52 @@
+"""
+Component for the swedish weather institute (SMHI) API.
+
+The location have
+to be at a coordinate within the Sweden borders.
+
+Data provided by SMHI is licenced under
+Creative Commons Attribution 4.0 International License
+
+Terms in swedish at:
+http://www.smhi.se/klimatdata/oppna-data/information-om-oppna-data/villkor-for-anvandning-1.30622
+
+This is implmenented as component to be able to use the configuration flow
+style of configuration. Implementations of the weather sensor are at
+homeassistant.components.weather.smhi
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components.weather.smhi/
+"""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import Config, HomeAssistant
+
+# Have to import for config_flow to work
+# even if they are not used here
+from .config_flow import smhi_locations  # noqa: F401
```

Comment:
Imported but not used

Reply:
Problem is that if I not use it the component wont load even if I dont use it here. So in order for the config_flow to work I have to make the exception. I also documented this above